src/input.h

```diff
@@ -5,6 +5,14 @@
 #define TIME_MILLIS(step_time) (step_time * 1000)
 #define TIME_SECONDS(step_time) (step_time * 1000000)
 #define MAX_LENGTH 128
+
+/* 
+ * Variable attenuation steps according to Vaunix LDA Linux SDK
+ * Attenuation setting in 0.25dB, so 4 steps = 1 dB
+ * Attenuation setting in 0.05dB, so 20 steps = 1 dB
+ * */
+#define MULTIPLIER_STEP 20
+
```

Comment:
It would be great, if you could automate this and make it based on the libversion or so. 
Do you know in which version the stepsize changed?

Reply:
Hi,
unfortunately I don't know from what version the stepsize changed.
I know that for the last available version (LDA097) the stepsize is 0.05dB (also if the API documentation says 0.25dB).

Reply:
@thuehn contacted Vaunix for clarification, so far it seems like version LDA097 is the version where the change happened.